examples/ingress/web.yaml

```diff
@@ -13,9 +13,18 @@ containers:
     value: wordpress
 services:
 - name: wordpress
-  type: LoadBalancer
   ports:
   - port: 8080
     targetPort: 80
+    endpoint: minikube.local/foo
+ingress:
+- name: root-ingress
```

Comment:
One defined here and one defined that gets auto-generated from the service with `endpoint`?